main_windows.go

```diff
@@ -0,0 +1,37 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+func appNeedsRootlessParentMain(clicontext *cli.Context) bool {
+	return false
+}
+
+func appBashComplete(clicontext *cli.Context) {
+	return
+}
+
+func bashCompleteNamespaceNames(clicontext *cli.Context) {
+	return
+}
+
+func bashCompleteSnapshotterNames(clicontext *cli.Context) {
+	return
+}
```

Comment:
Is there a reason to disable bash completion output on Windows builds? Or is this due to lack of Windows support in the nerdctl packages the bash-completion functions use?

Reply:
These functions wouldn't compile due to assumption for Linux. I didn't think that bash completion would function Windows so left them blank.

Reply:
Fair 'nuff for now. We can come back to it later. Most of my team uses bash on Window as their default shell (which surprised me when I joined them, and brings a bunch of its own issues), but without this PR, we don't have bash completion output on Windows anyway, so it's not a regression.

Reply:
> Most of my team uses bash on Window as their default shell 

WSL bash? Cygwin bash? MinGW bash?

Reply:
Git Bash, I believe, so Cygwin-derived via MSYS2, but has a different env-var for disabling `/`-path conversion.